LuhnAlgorithm/src/main/java/com/exercism/luhn/Mod10Validator.java

```diff
@@ -0,0 +1,84 @@
+package com.exercism.luhn;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Mod10Validator {
+    private static final String FILE_NAME = "TestData.txt";
+    private ArrayList<String> listToValidate;
+    Scanner scanner;
+
+    public Mod10Validator() {
+        try {
+            ClassLoader classLoader = getClass().getClassLoader();
+            File file = new File(classLoader.getResource(FILE_NAME).getFile());
+            scanner = new Scanner(file);
+            listToValidate = new ArrayList<String>();
+            System.out.println("Starting read from "+file.getName().toString());
+        } catch (FileNotFoundException e) {
+            System.out.println(FILE_NAME + " file not found.");
+            e.printStackTrace();
+        }
+    }
+
+    public void fileReader() {
```

Comment:
A good idea is to split the methods by responsibility. For instance, `fileReader` can be extracted to a `FileUtils` class and can be made static